pkg/cloudprovider/cloud.go

```diff
@@ -82,18 +80,18 @@ type LoadBalancer interface {
 	// TODO: Break this up into different interfaces (LB, etc) when we have more than one type of service
 	// GetLoadBalancer returns whether the specified load balancer exists, and
 	// if so, what its status is.
```

Comment:
My kingdom for a const pointer.

On Thu, Feb 18, 2016 at 10:59 AM, Daniel Smith <notifications@github.com>
wrote:

> In pkg/cloudprovider/cloud.go
> <https://github.com/kubernetes/kubernetes/pull/21378#discussion_r53363515>
> :
>
> > @@ -82,18 +80,18 @@ type LoadBalancer interface {
> >  	// TODO: Break this up into different interfaces (LB, etc) when we have more than one type of service
> >  	// GetLoadBalancer returns whether the specified load balancer exists, and
> >  	// if so, what its status is.
>
> Make it clear in the comments that the implementing functions are *not*
> allowed to change the service. (Or, if they do, they should not expect
> those changes to be persisted.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/21378/files#r53363515>.
>